src/sass/modern_style.scss

```diff
@@ -72,6 +72,27 @@ hr.small {
     top: 5px;
     bottom: auto;
   }
+
```

Comment:
Not needed in this file, we have the `@import 'classic_style';` which will include it. Only needed here if we want to apply a different styling.

Reply:
Thanks! Didn't saw the import :)